recipes/python-3.5/meta.yaml

```diff
@@ -0,0 +1,54 @@
+package:
+  name: python
+  version: 3.5.1
+
+source:
+  fn: Python-3.5.1.tgz
+  url: https://www.python.org/ftp/python/3.5.1/Python-3.5.1.tgz
+  md5: be78e48cdfc1a7ad90efff146dce6cfe
+  patches:
+    - osx-dist.patch       # [osx]
+    - win-find_exe.patch   # [win]
+
+build:
+  no_link: 
+    - bin/python
+  # Skip windows build for the time being
+  skip: True   # [win]
+  number: 0
+
+requirements:
+  build:
+    - bzip2     # [unix]
+    - zlib      # [unix]
+    - sqlite    # [unix]
+    - readline  # [unix]
+    - tk        # [unix]
+    - openssl   # [unix]
+    - xz        # [unix]
+    - ncurses   # [linux]
+  run:
+    - zlib      # [unix]
+    - sqlite    # [unix]
+    - readline  # [unix]
+    - tk        # [unix]
+    - openssl   # [unix]
+    - xz        # [unix]
+    - ncurses   # [linux]
+
+test:
+  commands:
+    - python -V             # [unix]
+    - python3 -V            # [unix]
+    - 2to3 -h
+    - pydoc -h
+    - python3-config --help  # [unix]
+
+about:
+  home: http://www.python.org/
+  license: PSF
+  summary: general purpose programming language
+      
+extra:
+  recipe-maintainers:
+    - jjhelmus
```

Comment:
Please add me as a maintainer on this. Additionally, I suspect @jakirkham and @msarahan may want to be added too.